great_tables/gt.py

```diff
@@ -403,17 +404,150 @@ def _get_column_of_values(gt: GT, column_name: str, context: str) -> List[str]:
     return cell_values
 
 
-def _as_raw_html(gt: GT) -> str:
+def as_raw_html(gt: GT) -> str:
     """
-    Returns the GTData object as raw HTML.
+    Get the HTML content of a GT object.
 
-    Args:
-        gt (GT): The GTData object to convert to raw HTML.
-        context (str): The context in which to build the output.
+    Get the HTML content from a GT object as a string. This function is useful for obtaining the
+    HTML content of a GT object for use in other contexts.
 
-    Returns:
-        str: The GTData object as raw HTML.
+    Parameters
+    ----------
+    gt
+        A GT object.
+
+    Returns
+    -------
+    str
+        An HTML fragment containing a table.
     """
     gt_built = gt._build_data(context="html")
     html_table = gt_built._render_as_html()
     return html_table
+
+
+def gtsave(
```

Comment:
Can we move this logic out of `gt.py`? It seems helpful to restrict this file to the creation of the GT class (which is mostly a wrapper around other things)

Reply:
Putting all of this into `_export.py`.